config/samples/uperf-scale/cr.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
+kind: Benchmark
+metadata:
+  name: uperf-scale-benchmark
+  namespace: benchmark-operator
+spec:
+  clustername: myk8scluster
+  elasticsearch:
+    url: https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com:443
+  #test_user: username_to_attach_to_metadata
+  workload:
+    # cleanup: true
+    name: uperf-scale
+    args:
+      serviceip: false
+      hostnetwork: false
+      networkpolicy: false
+      multus:
+        enabled: false
+      samples: 5
+      kind: pod
+      # pin: false
```

Comment:
It seems scale-uperf does not support pin mode. So remove its refences.  'pair' also.